### DISCUSSION TITLE
Why is Go Modules not being used?

### DISCUSSION THREAD

**Discussion by BrenekH** [BanDev/Lux#6] [General]
Go Modules is the official dependency management system of Go (introduced in Go 1.11 and made the default in Go 1.16). It allows for automatic download of dependencies and the use of the `go install` command to install a binary from a repository.

In addition, modules are super easy to set up. All it takes is `go mod init github.com/BanDev/Lux` and any dependencies are installed using `go get <repo url>`.

**Comment by jackdevey** [comment]
Hey, thanks for this! Lux is the first Go application I have written and I had no idea about this. I'll switch to Go Modules straight away!

**Reply by jackdevey** [reply]
I think I have done this now? [2fafb190e8f93cc3bfc8358fcd26a3f14a094951](https://github.com/BanDev/Lux/commit/2fafb190e8f93cc3bfc8358fcd26a3f14a094951)

**Reply by BrenekH** [reply]
Looks great!
